### ISSUE TITLE
Can't save a request without the name

### ISSUE THREAD

**Issue by Wadjetz** [usebruno/bruno#1702]
When you try to save a request you can't save it without name of the request.

it would be nice to have a better message error

<img width="280" alt="Capture d’écran 2024-03-01 à 15 26 41" src="https://github.com/usebruno/bruno/assets/2094791/2c482736-b4fb-4d9d-9116-756e7c922fc6">
<img width="362" alt="Capture d’écran 2024-03-01 à 15 32 15" src="https://github.com/usebruno/bruno/assets/2094791/42621bfd-eee7-4210-9d5f-fa2358d54164">

**Comment by sanjai0py** [comment]
Hey there, @Wadjetz! Can you provide me with more information, please? I noticed that in Bruno `1.10.0`, you can not create a request without a name. Is that what you were looking for, or did I miss something? 

![image](https://github.com/usebruno/bruno/assets/84461672/9b317ed0-aaf2-49a2-a70b-0d31b66129a5)

**Comment by Wadjetz** [comment]
Yes, I used Bruno for the first time and created a request from the file using the example from the documentation.
`get {
  url: https://api.github.com/users/usebruno
}`
And then I wanted to add a header or modify it from the desktop app (v1.10.0), but encountered this error.

**Comment by sreelakshmi-bruno** [comment]
Closing this ticket since this issue has been fixed. Please reopen if you find otherwise, thanks!
